float32 radius
float32 maxSpeed
float32 maxAcc
int64 maxNeighbors
float32 neighborDist
float32 timeHorizon
float32 timeHorizonObst
float32 lambdaValue
int32 agentType
int32 NORMAL=0
int32 AGGRESSIVE=1
int32 USER=2
int32 USER_PREF=3
int32 USER_ADAPTIVE=4
int32 DYN_OBS=4
int32[] topoKeys
int32[] topoValues
int32[] lambdaKeys
int32[] lambdaValues
int32 TOPO_RVO=0
int32 TOPO_HEAD=1 
int32 TOPO_LEFT=2
int32 TOPO_RIGHT=3